systems/subscriber/registry/pkg/config.go

```diff
@@ -26,7 +26,7 @@ func NewConfig(name string) *Config {
 		Service: uconf.LoadServiceHostConfig(name),
 		MsgClient: &uconf.MsgClient{
 			Timeout:        7 * time.Second,
-			ListenerRoutes: []string{""},
+			ListenerRoutes: nil,
```

Comment:
Why giving a nil ListenerRoutes if it is not used?

Reply:
addressed